extensions/arc/package.json

```diff
@@ -58,19 +58,31 @@
       "title": "Open New Tab",
       "subtitle": "Arc",
       "description": "Create and open a new tab in the current window",
-      "mode": "no-view"
+      "mode": "no-view",
+      "preferences": [
+        {
+          "name": "url",
+          "title": "URL",
+          "description": "The URL to open in the new tab",
+          "type": "textfield",
+          "required": false,
+          "default": "https://google.com"
+        }
+      ]
     }
   ],
   "dependencies": {
     "@raycast/api": "^1.45.0",
     "@raycast/utils": "^1.4.0",
+    "is-url": "^1.2.4",
```

Comment:
do you think it's necessary to use a library for this? it seems like a fairly simple [regex](https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch08s01.html) would do the trick.

Reply:
I checked the package and it does pretty much that, this is why I pulled it in but good research.